### DISCUSSION TITLE
Walbash has stopped working

### DISCUSSION THREAD

**Discussion by 4thehalibit** [prasanthrangan/hyprdots#1148] [Q&A] [answered]
I have ran some commands but not sure if I am doing it correctly 
```
rm -fr ~/.config/hypr/scripts  
rm -fr ~/.config/hypr/wallbash 
```

then when I run this: `Hyperdots rebuild cache` it returns 
```
Please run:
      'Hyprdots-install'                                 Install Hyprdots (default)
For advanced usage see options below
      --clone= /path/to/Clone/Hyprdots                      Clone Hyprdots in a Custom path then run installation.
                                                                Default: '/home/vwestberg/Hyprdots'
      --git https://gitclone/link/                          Repository link to be cloned
                                                                Default: 'https://github.com/prasanthrangan/hyprdots'

```
![image](https://github.com/prasanthrangan/hyprdots/assets/12929552/26aa04e3-1d77-4718-a96d-37fc5197d061)

**Comment by kRHYME7** [comment]
Hello please update first the The Hyprodts cli  ``` yay -Sy hyprdots-ctl-git ```

**Reply by 4thehalibit** [reply]
I have updated the Hyprdots and rebuilt cache

**Reply by kRHYME7** [reply]
is it fixed now ?

**Reply by 4thehalibit** [reply, marked as answer]
Yes it is working now. Although I have restarted I had to change theme and go back to it
Solution was:
1. Run `yay -Sy hyprdots-ctl-git` which gave error because yay was borked fixed with `sudo ln -s /usr/lib/libalpm.so.14.0.0 /usr/lib/libalpm.so.13`
2. Re run `yay -Sy hyprdots-ctl-git`
3. rebuilt cache `Hyperdots rebuild cache`

**_IMPORTANT_** 
**Switch themes even a restart wont help**
